apps/wallet/src/ui/app/pages/accounts/manage/AccountGroupItem.tsx

```diff
@@ -173,6 +177,11 @@ export function AccountGroupItem({
                                         Export Private Key
                                     </ListItem>
                                 ) : null}
+                                {account.isKeyPairExportable ? (
```

Comment:
It should also be possible to export the public key from a ledger account, one can get the public key from a ledger nano